opensearch/src/main/java/org/opensearch/sql/opensearch/util/OpenSearchRelOptUtil.java

```diff
@@ -30,6 +33,9 @@
 import org.apache.calcite.util.mapping.Mappings;
 import org.apache.commons.lang3.tuple.Pair;
 import org.opensearch.sql.calcite.utils.OpenSearchTypeFactory;
+import org.opensearch.sql.opensearch.storage.scan.AbstractCalciteIndexScan;
+import org.opensearch.sql.opensearch.storage.scan.context.PushDownType;
+import org.opensearch.sql.opensearch.storage.scan.context.SortExpressionInfo;
 
 @UtilityClass
 public class OpenSearchRelOptUtil {
```

Comment:
What is `OpenSearchRelOptUtil` used for?  I don't see any reason to keep it in opensearch module. Can this be merged to `PlanUtlis`?